components/Radio/src/stories/bem.stories.mdx

```diff
@@ -98,9 +143,9 @@ import readme from "../../README.md";
       <label class="denhaag-form-control-label">
```

Comment:
{() => (
Volgens mij mag de react uit de bem, hiervoor is een aparte react.stories.mdx file